.github/workflows/test.yml

```diff
@@ -57,59 +57,47 @@ jobs:
 
     - name: apt update
       run: |
-        sudo apt update
+        apt-get update
 
     - name: Verify no uncommitted changes from make lint
       run: |
         git init
         git add .
         chown -R testbot:testbot .
-        sudo -u testbot bash -lc 'make lint-setup lint'
-        GEN_DIFF=$(git status -s)
+        sudo -u testbot bash -lc 'sudo apt-get install -y clang-format && make lint'
```

Comment:
[q] Did you remove the brew install as these ymls are only run in linux based CI?